RockLib.Configuration/RockLibConfigurationBuilderExtensions.cs

```diff
@@ -38,7 +42,7 @@ public static IConfigurationBuilder AddRockLib(this IConfigurationBuilder builde
         /// <exception cref="ArgumentNullException">If <paramref name="builder"/> is null.</exception>
         /// <exception cref="ArgumentException">If <paramref name="jsonConfigPath"/> is null or empty.</exception>
         /// <returns>The <see cref="IConfigurationBuilder"/>.</returns>
-        public static IConfigurationBuilder AddRockLib(this IConfigurationBuilder builder, string jsonConfigPath, string supplementalJsonConfigPath = null)
+        public static IConfigurationBuilder AddJson(this IConfigurationBuilder builder, string jsonConfigPath, string supplementalJsonConfigPath = null)
```

Comment:
Does this method provide any value anymore? Should it be made private?

Reply:
Not really.  I'll clean it up, thanks.